pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/vedlegg/opplysningerbruktiberegningufoere/OpplysningerOmBarnetillegg.kt

```diff
@@ -0,0 +1,986 @@
+                    Nynorsk to "For ".expr() +
+                            ifElse(harTilleggForFlereSaerkullsbarn, "barna", "barnet")
+                            + " som ikkje bur saman med begge foreldra, fastset vi storleiken på barnetillegget ut frå inntekta di. Inntekt til ein ektefelle/partnar/sambuar som ikkje er forelder til barnet, har ikkje betydning for storleiken på barnetillegget. Barnetillegget blir redusert dersom den samla inntekta di er høgare enn fribeløpet. Fribeløpet for eit barn som ikkje bur saman med begge foreldra, er 3,1 gonger grunnbeløpet i folketrygda, og aukar med 40 prosent av grunnbeløpet i folketrygda for kvart ekstra barn.".expr(),
+                    English to "For the ".expr() +
+                            ifElse(harTilleggForFlereSaerkullsbarn, "children who do", "child who does")
+                            + " not live together with both parents, the amount of child supplement is based on your income. The income of a spouse/partner/cohabitant who is not the child's parent, is not taken into consideration. The child supplement will be reduced if your total income is greater than the exemption amount. The exemption amount is 3.1 times the National Insurance basic amount and it increases with 40 percent of the National Insurance basic amount for each extra child.".expr(),
+                )
+            }
+            showIf(harAnvendtTrygdetidUnder40 and not(harYrkesskade)) {
+                paragraph {
+                    text(
+                        Bokmal to " Siden trygdetiden din er kortere enn 40 år, blir fribeløpet redusert ut fra den trygdetiden du har.",
+                        Nynorsk to " Sidan trygdetida di er kortare enn 40 år, blir fribeløpet redusert ut frå den trygdetida du har.",
+                        English to " Since you have less than 40 years National Insurance membership, the exemption amount is ruduced correspondingly in relation to how long you have had National Insurance membership."
+                    )
+                }
+            }
+        }
+    }
+
+    // TBU605V
+    data class PeriodisertInntektInnledning(
+        val sivilstand: Expression<Sivilstand>,
+        val harKravaarsakEndringInntekt: Expression<Boolean>,
+        val harFlereTillegg: Expression<Boolean>,
+    ) : OutlinePhrase<LangBokmalNynorskEnglish>() {
+        override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+            paragraph {
+
+                val foreldreBorSammen =
```

Comment:
Lurer på om denne her ville lest mye bedre som en showIf på sivilstand og en textExpr med fullstendig setning for de to tilfellene.

Reply:
Det ser bedre ut ja. Endrer.

Reply:
endret.